modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -12,6 +12,7 @@ processes (number of processes instances, number of current users, operations an
 
 Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Portal)
 
+[width="30%"]
```

Comment:
`30%` seems too small

**With 30%**
![image](https://user-images.githubusercontent.com/27200110/110305878-7a492700-7ffd-11eb-9e73-808851c79895.png)

**With default**
![image](https://user-images.githubusercontent.com/27200110/110306010-9f3d9a00-7ffd-11eb-9efb-69f54cc1f104.png)

Reply:
ok thanks. I think that 50% would be nice. I'm trying that.